### PR TITLE
Clarify the LLVM dependency on Windows

### PR DESCRIPTION
**Pull request by weazeltech**
I had some trouble understanding that I had to install LLVM separately and it wasn't included with the Visual Studio as a dependency. I'm creating this PR because it confused me and I saw some people on the discord go through this as well, and that's how I "figured it out myself".

### CODE REVIEW

File: src/gdnative/intro/setup.md [hunk 1]
```diff
@@ -49,4 +49,4 @@ clang -v
 
 One way to get started with godot-rust is a full-fledged (inofficial) template, which can be found [here](https://github.com/macalimlim/godot-rust-template) to get you started right away. All the boilerplate stuff is already done for you, however, using the template requires you to set up extra dependencies and toolchains. Check out the [wiki](https://github.com/macalimlim/godot-rust-template/wiki) for instructions on how to get started with the template.
 
-The template is not maintained by us, and might not work in all setups where the base library would be compatible. If you encounter any issues with the template, please report them at its [issue tracker](https://github.com/macalimlim/godot-rust-template/issues/).
\ No newline at end of file
+The template is not maintained by us, and might not work in all setups where the base library would be compatible. If you encounter any issues with the template, please report them at its [issue tracker](https://github.com/macalimlim/godot-rust-template/issues/).
```
Comment (Bromeon) [inline src/gdnative/intro/setup.md:52]: Any idea what changed here? Line ending? Empty line?

File: src/gdnative/intro/setup.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ rustc --version
 cargo -V
 ```
 
-### Windows
+## Windows
 
-When working on Windows, it's also necessary to install the [Visual Studio Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/), or the full Visual Studio (*not* Visual Studio Code). More details can be found on [Working with Rust on Windows](https://github.com/rust-lang/rustup#working-with-rust-on-windows).
+When working on Windows, it's also necessary to install the [Visual Studio Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/), or the full Visual Studio (*not* Visual Studio Code). More details can be found on [Working with Rust on Windows](https://github.com/rust-lang/rustup#working-with-rust-on-windows). Also note that LLVM is also required on top of those dependencies to be able to build your godot-rust project, see the next section for more information.
```
Comment (Bromeon) [inline src/gdnative/intro/setup.md]: I wouldn't repeat "also", and avoid "to be able to":

> Note that LLVM is also required on top of those dependencies to build your godot-rust project, see the next section for more information.

**Review by Bromeon — COMMENTED** [review]
Thanks for clarifying this! 👍

File: src/gdnative/intro/setup.md (hunk 1 above)
Comment (weazeltech) [reply src/gdnative/intro/setup.md:52]: I edited it directly on github, so I really don't know what triggered this issue, my guess is that it is changing the end of file character somehow, even though I didn't touch it.

File: src/gdnative/intro/setup.md (hunk 2 above)
Comment (weazeltech) [reply src/gdnative/intro/setup.md]: Thanks for the feedback, will submit a fix

**Comment by Bromeon** [comment]
Thanks a lot!

**Pull request merged**